splice_machine/src/main/java/com/splicemachine/procedures/external/GetSchemaExternalResult.java

```diff
@@ -48,9 +52,14 @@ public boolean isSuccess(){
         return true;
     }
 
-    public String getSuggestedSchema()
+    public String getSuggestedSchema(String separator)
     {
-        return ExternalTableUtils.getSuggestedSchema(schema, partition_schema);
+        String s = ExternalTableUtils.getSuggestedSchema(schema, partition_schema, separator);
```

Comment:
I would use a `StringBuilder` for this (probably IntelliJ can automatically do that for you)

Reply:
done